docs/notebooks/getting_started/units.ipynb

```diff
@@ -0,0 +1,823 @@
+   "execution_count": null,
+   "id": "21c46ef1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity.to(u.m / u.s)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1e742ca",
+   "metadata": {},
+   "source": [
+    "[Quantity]: https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity\n",
+    "[si]: https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity.si\n",
+    "[cgs]: https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity.cgs\n",
+    "\n",
+    "The [si] and [cgs] attributes convert the [Quantity] to SI or CGS units, respectively. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "02bda4d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity.si"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "560a0cfe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity.cgs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1429d5a2",
+   "metadata": {},
+   "source": [
+    "## Detaching units and values\n",
+    "\n",
+    "[value]: https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity.value \n",
+    "[Quantity]: https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity\n",
+    "\n",
+    "The [value] attribute of a [Quantity] provides the number or array without the unit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24e8403c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "time.value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c6c027b",
+   "metadata": {},
+   "source": [
+    "[unit]: https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity.unit\n",
+    "[Quantity]: https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity\n",
+    "\n",
+    "The [unit] attribute of a [Quantity] provides the unit without the value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2c41a2fa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "time.unit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be0c5349",
+   "metadata": {},
+   "source": [
+    "## Equivalencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07c3b9c1",
+   "metadata": {},
+   "source": [
+    "[electron-volt]: https://en.wikipedia.org/wiki/Electronvolt\n",
+    "[Boltzmann constant]: https://en.wikipedia.org/wiki/Boltzmann_constant\n",
+    "\n",
+    "Plasma scientists often the [electron-volt] (eV) as a unit of temperature. This is a shortcut for describing the thermal energy per particle, or more accurately the temperature multiplied by the [Boltzmann constant], $k_B$. Because an electron-volt is a unit of energy rather than temperature, we cannnot directly convert electron-volts to kelvin."
```

Comment:
```suggestion
    "Plasma scientists often use the [electron-volt] (eV) as a unit of temperature. This is a shortcut for describing the thermal energy per particle, or more accurately the temperature multiplied by the [Boltzmann constant], $k_B$. Because an electron-volt is a unit of energy rather than temperature, we cannot directly convert electron-volts to kelvin."
```